### PR TITLE
Add example

### PR DESCRIPTION
**Pull request by YPares**
Hi :) I added a small example use case

### CODE REVIEW

File: example/wedding.nu [hunk 1]
```diff
@@ -0,0 +1,31 @@
+def main [file: path] {
```
Comment (PhotonBursted) [inline example/wedding.nu]: Should we add a default here? Perhaps even hard-code the path to the CSV?

File: example/wedding.nu [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        $tasks | each {|task|
+            $task_ds | get $task | {task: $task, magnitude: ($in | vec magnitude)}
+        } |
+        sort-by magnitude | last 3 |
+        table
+    )
+    print "Seems that DJ-ing is the most staffed task. Wait... isn't it a one-person job normally?"
+
+    let name = "Maya"
+    print $"Who tends to work on the same tasks than ($name)?"
```
Comment (PhotonBursted) [inline example/wedding.nu]: as*

File: example/wedding.nu [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    let people_ds = $whole_ds | reject task
+    let task_ds = $whole_ds | transpose -r
+    
+    let tasks = $task_ds | columns
+    print "Which are the 3 most staffed tasks?"
+    print (
+        $tasks | each {|task|
+            $task_ds | get $task | {task: $task, magnitude: ($in | vec magnitude)}
+        } |
+        sort-by magnitude | last 3 |
```
Comment (PhotonBursted) [inline example/wedding.nu]: Perhaps we could sort in reverse, so the top one is at the top of the list (would be a bit more intuitive to me)

File: example/wedding.nu [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        table
+    )
+    print "Seems that DJ-ing is the most staffed task. Wait... isn't it a one-person job normally?"
+
+    let name = "Maya"
```
Comment (PhotonBursted) [inline example/wedding.nu]: Maya results in three exactly the same similarity results. Do we have a name which leads to results that are a little more "interesting"?

**Review by PhotonBursted — COMMENTED** [review]
Great idea! Thanks for the addition ^~^

I've added a few comments to this, see what you think of them.

**Comment by YPares** [comment]
Done :)

**Pull request merged**